src/routes/CreateMarcBibRoute/CreateMarcBibRoute.js

```diff
@@ -19,14 +25,22 @@ const QuickMarcRoute = ({ match, history, location }) => {
     });
   }, [location.search]);
 
+  const onCreateAndKeepEditing = useCallback((id) => {
+    history.push(`edit-bibliographic/${id}`);
+  }, []);
+
   return (
     <div data-test-inventory-quick-marc>
       <Pluggable
         type="quick-marc"
-        basePath={match.path}
         onClose={onClose}
         onSave={onClose}
         externalRecordPath="/inventory/view"
+        action="create"
+        marcType="bibliographic"
+        isShared={searchParams.get('shared')}
```

Comment:
Should `isShared` be a string?

Reply:
[Yes, it should be](https://github.com/folio-org/ui-quick-marc/blob/704e3421d79e931bacbfa87dfc4316c984257158/src/QuickMarc.js#L154). Good catch, @Dmytro-Melnyshyn!

Other route components define
```
  const isShared = searchParams.get('shared') === 'true';
```
... but then pass `isShared={searchParams.get('shared')}`, which feels like a good. I think all should define that value and then use it in the props to the plugin.